tests/test_contrib_reversion.py

```diff
@@ -156,13 +156,12 @@ def test_revision_is_visible_in_list():
 
```

Comment:
test_revert_to_previous_version doesn't need perms?
besides that maybe some user_without_perms-tests would be nice

Reply:
Good catch; I'll look into that. Also I'll add no-permission tests for all views